lib/core/utils/get-bounding-size.js

```diff
@@ -0,0 +1,11 @@
+export default function getBoundingSize(domNode) {
+  let width = parseInt(domNode.getAttribute('width'), 10);
+  let height = parseInt(domNode.getAttribute('height'), 10);
+
+  if (isNaN(width) || isNaN(height)) {
+    const rect = domNode.getBoundingClientRect();
+    width = isNaN(width) ? rect.width : width;
```

Comment:
This will accept any element with `width` or `height` attribute as the true size, even if the element itself doesn't apply them. Since this is a util function we'll need to account for non-frame elements.

```js
fixture.innerHTML = '<div width="100px"></div>';
var node = fixture.querySelector('div');
axe.utils.getBoundingSize(node); // { width: 100, height: 0 }
```

Reply:
I didn't put it on axe.utils. I don't think that matters.

Reply:
You did https://github.com/dequelabs/axe-core/pull/3047/files#diff-b5e51e5519d1527b32e31b86be8881c77bc4aed33ebcb09361127b17376ff6f7R30